mappings/com/mojang/blaze3d/texture/TextureUtil.mapping

```diff
@@ -39,3 +42,11 @@ CLASS com/mojang/blaze3d/platform/TextureUtil com/mojang/blaze3d/texture/Texture
 		ARG 3 mipLevels
 		ARG 4 width
 		ARG 5 height
+	METHOD writeAsPNG (Ljava/nio/file/Path;Ljava/lang/String;IIIILjava/util/function/IntUnaryOperator;)V
```

Comment:
Do we use `PNG` or `Png`? I feel like we should be using `Png`

Reply:
This was automapped and matches the above function. for now i say we keep it the same